loveship/components/overlays/NotificationCard.tsx

```diff
@@ -1,88 +1,37 @@
-    const notificationCardNode = React.useRef(null);
+export interface NotificationCardProps extends Omit<uuiNotificationCardProps, 'color' >, NotificationCardMods {}
 
-    React.useImperativeHandle(ref, () => notificationCardNode.current, [notificationCardNode.current]);
 
-    useLayoutEffectSafeForSsr(() => {
-        notificationCardNode.current?.addEventListener('mouseenter', props.clearTimer);
-        notificationCardNode.current?.addEventListener('mouseleave', props.refreshTimer);
-        return () => {
-            notificationCardNode.current?.removeEventListener('mouseenter', props.clearTimer);
-            notificationCardNode.current?.removeEventListener('mouseleave', props.refreshTimer);
-        };
-    }, []);
-
-    return (
-        <div
-            role="alert"
-            className={ cx(css.notificationWrapper, styles[`color-${ props.color }`], props.cx) }
-            ref={ notificationCardNode }
-            { ...props.rawProps }
-        >
-            <div className={ css.mainPath }>
-                { props.icon && (
-                    <div className={ css.iconWrapper }>
-                        <IconContainer icon={ props.icon } cx={ css.actionIcon } />
-                    </div>
-                ) }
-                <div className={ css.content }>
-                    { props.children }
-                    { props.actions && <div className={ css.actionWrapper }>
-                        { props.actions.map(action => (
-                            <LinkButton
-                                caption={ action.name }
-                                onClick={ action.action }
-                                key={ action.name }
-                                cx={ css.actionLink }
-                                size='36'
-                                rawProps={ action.rawProps }
-                            />
-                        )) }
-                    </div> }
-                </div>
-                { props.onClose && (
-                    <div className={ css.closeWrapper }>
-                        <IconButton
-                            icon={ CrossIcon }
-                            color='night600'
-                            onClick={ props.onClose }
-                            cx={ css.closeIcon }
-                        />
-                    </div>
-                ) }
-            </div>
-        </div>
-    );
-});
+export const NotificationCard = withMods<Omit<uuiNotificationCardProps, 'color' >, NotificationCardMods>(
+    uuiNotificationCard,
+    applyNotificationCardMods,
+    (props) => ({
+        ...props,
+        color: props.color ?? 'night600',
+    }),
+);
 
 export const WarningNotification = React.forwardRef<HTMLDivElement, DefaultNotificationProps>((props, ref) => (
     <NotificationCard icon={ WarningIcon } color='sun' ref={ ref } { ...props } cx={ cx(props.cx, css.typeWarning) } />
```

Comment:
we removed css.typeWarning class, so remove this classes from code

Reply:
Done